website/core/Footer.js

```diff
@@ -51,11 +51,11 @@ class Footer extends React.Component {
               User Showcase
             </a>
             <a
-              href="http://stackoverflow.com/questions/tagged/"
+              href="http://stackoverflow.com/questions/tagged/unibeautify"
```

Comment:
I think StackOverflow tags need to be explicitly added to their service. May need sufficient reputation to do so.

Reply:
I'd like to start encouraging users to use Stack Overflow first 👍 .